### DISCUSSION TITLE
Quick automated plot generation from PK-sim

### DISCUSSION THREAD

**Discussion by nguyen-tranbach** [Open-Systems-Pharmacology/Forum#1865] [How-Tos]
Dear community,

I actually have finished running a parameter identification containing 30 individual simulations, which correspond each to around 10-15 kinetics in fluids and tissues.

How can I quickly generate goodness-of-fit plots **with confidence/prediction intervals** for each kinetics (VPC, individual fit by fluid and tissue with CI/PI) ?

Unfortunately, I have no idea how to transfer this to R or Matlab (not a competent user of R or Matlab either).

Thank you so much for your help !

**Comment by StephanSchaller** [comment]
Hi @nguyen-tranbach,

have you activated "calculate sensitivities" for the parameter identification:

![image](https://github.com/user-attachments/assets/804bf637-e57c-4cb0-b98c-08907bf27587)

This will enable the VPC graphs.

**Reply by nguyen-tranbach** [reply]
Dear Stephan,
Yes I did. The problem is that it will take to long to generate plots by individual and by compartment, and I do not know how to improve this, as I have many other models to test with the same plots to generate

**Reply by StephanSchaller** [reply]
Hi @nguyen-tranbach , not sure I understand. These plots are generated per PI set-up. Usually individuals are pooled if they have undergone the same study protocol. Can you provide an example of what you are trying to achieve?

Otherwise, if customized, such a solution would have to be (and can be) created through R.

**Reply by nguyen-tranbach** [reply]
You can find my project [here](http://iame.catibiomed.fr/index.php/s/pXHFqFwPjYmvOgv), in which all my questions are involved actually

**Reply by nguyen-tranbach** [reply]
I don't know much about your R package. Maybe I did it wrong but I found it impossible to export the VPC plot data to Excel, which I reported in another post
